.pre-commit-config.yaml

```diff
@@ -44,7 +50,7 @@ repos:
           - -d duplicate-code
 
   - repo: https://github.com/adrienverge/yamllint.git
-    rev: 'v1.23.0'
+    rev: "v1.23.0"
```

Comment:
Maybe let's take the opportunity in this PR to give nice names to all the hooks when running `pre-commit`:

```
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check for added large files..............................................Passed
Check for merge conflicts................................................Passed
isort....................................................................Passed
black....................................................................Passed
Check for PEP8 error on Python files.....................................Passed
Check for Linting error on Python files..................................Passed
Check for Linting error on YAML files....................................Passed
Check for Linting error on Jinja2 templates..............................Passed
markdownlint.............................................................Passed
```

So for isort, black and markdownlint ?

Reply:
done